backends-clickhouse/src/main/scala/org/apache/spark/sql/delta/ClickhouseOptimisticTransaction.scala

```diff
@@ -119,6 +120,16 @@ class ClickhouseOptimisticTransaction(
           }.toMap
       }
 
+      spark.conf.getAll.foreach(
+        entry => {
+          if (
+            entry._1.startsWith(s"${CHBackendSettings.getBackendConfigPrefix}.runtime_settings")
+            || entry._1.equalsIgnoreCase(DeltaSQLConf.DELTA_OPTIMIZE_MAX_FILE_SIZE.key)
```

Comment:
it's better use the 'spark.databricks.delta.optimize.minFileSize' to control the size which will be merged.

Reply:
fixed